docs/styleguide/PyGuide.md

```diff
@@ -122,6 +122,18 @@ Run [pylint](https://github.com/PyCQA/pylint) over your code using this [pylintr
 - Wildcard imports (`from module import *`) should be avoided, as they make it unclear which names are present 
   in the namespace, confusing both readers and many automated tools.
 
+- For classes from the typing module. You are explicitly allowed to import multiple specific classes on one line from the typing module.
+  ```python
+  # Correct:
+  from typing import Any, Dict, Optional
+  ```
+  ```python
+  # Wrong:
+  from typing import Any 
+  from typing import Dict
+  from typing import Optional
+  ```
```

Comment:
The IDE will probably only automatically adhere to one variant or the other, so this requirement is contradictory to the requirement to have separate lines for all other imports.

Reply:
We try to follow the Google Styleguide for Python. PyTorch and TensorFlow frameworks support it. The proposed approach is used within the frameworks supported by the NNCF and has worked well (IMHO: fewer lines and more readability).